commands/nasa.js

```diff
@@ -14,14 +14,25 @@ exports.run = async (client,interaction) => {
     let res = await fetch(`https://api.nasa.gov/planetary/apod?api_key=${client.config.nasa_token}`);
     let jsonData = await res.json();
     console.log(jsonData);
-    const embed = new MessageEmbed()
+    if(jsonData.media_type == 'video') {
```

Comment:
We should cut down on the duplicated code here -- create one messageembed, set the common options, then have an if statement to set the description length and author if needed